web/styles/left_sidebar.css

```diff
@@ -450,14 +450,16 @@
            to -0.12em produces a slightly more pleasing result, which makes sense
            because mathematical centering and the perception of center don't
            always match. */
-        margin-top: -0.12em;
```

Comment:
I'm no expert on this, but there's 10 lines of comment justifying this value. You've reset it to 0, and left the comments as-is. Is there really no other way?